### ISSUE TITLE
Simple request: The title.

### ISSUE THREAD

**Issue by coffeecrisp** [lpereira/hardinfo#586]
Hello, I run Kubuntu 18.04 and will soon move to 20.10.
The application name seems to be HardInfo but when I launch the program, in the title bar, it says System Information.
This is bizarre. The standard is to show the application's name.

Thank you!

**Comment by mckaygerhard** [comment]
not more bizarre than that M$ like linux.. is just a title and it points the right : system information.. the command is "hardinfo" and the title is "system information" cdos is a title just like a title in some presentation will differs from the name of the file..

**Comment by lpereira** [comment]
You're right.  I thought HardInfo was following the GNOME HIG as far as the naming goes, and it's apparently not completely: https://wiki.gnome.org/Design/HIG/ApplicationName

This is easy to change, though, and will be fixed in the next release:

- GenericName will be set to "System Information and Benchmark" in the desktop file
- Name will be HardInfo
- Title bar won't change, as the application name (HardInfo) will be shown in the top bar

**Comment by mckaygerhard** [comment]
> * Title bar won't change, as the application name (HardInfo) will be shown in the top bar

that the same to said "no"  XD

**Comment by lpereira** [comment]
Gerard, it's not just what you say, but how you say it. This isn't the
first time this has happened and it's not the first time I tell you that
this isn't acceptable behavior. I know you're even more excited about this
project than I am, but this is still my personal project and I expect
people to be polite here. (Please don't respond to this comment as this is
already off topic and I don't want to spend my time off open source with
stuff like this.)

**Comment by coffeecrisp** [comment]
I think I understand. In the GNOME desktop style, they have the bar at the top so I guess your link says that is where the app name will appear.

I am porting a couple of my apps to Linux so I may have to take that into account.

Mckaygerhard, I prefer the Win95 style in terms of positions, menus. I could even go back further, the Win 3.1  style. Win 8 and 10 lost its colors and too many changes. That is why I moved to Kubuntu.

**Comment by mckaygerhard** [comment]
the problem is that gnome breaks many things in desktop behaviour to provide their own "xperience" (xperience? hug?) ...

reading at https://specifications.freedesktop.org/wm-spec/latest/ar01s05.html i confirmed that there's no consencus about what must be in the title.. neither at https://specifications.freedesktop.org/wm-spec/latest/ar01s09.html .. so i guess you are on your own, sorry!

about "windoze" thigns.. winbuntu is crap .. no unix like behaviour..

**Comment by coffeecrisp** [comment]
"about "windoze" thigns.. winbuntu is crap .. no unix like behaviour.."

What do you mean? What is missing from winbuntu?
